apps/challenges/views.py

```diff
@@ -1840,6 +1841,7 @@ def download_all_submissions(
                             submission["created_at"],
                             submission["submission_result_file"],
                             submission["submission_metadata_file"],
+                            submission["submission_metadata"],
```

Comment:
@afif-fahim can we just fetch fields `name` and `value` for each attribute in submission metadata and just send that in this column

Reply:
@Ram81, can you review again please?
After changes:
![image](https://user-images.githubusercontent.com/33936462/109487660-b6850080-7aae-11eb-9ac3-c30e7a8c1c07.png)